examples/flutter_gallery/lib/demo/material/cards_demo.dart

```diff
@@ -9,18 +9,26 @@ import '../../gallery/demo.dart';
 
 const String _kGalleryAssetsPackage = 'flutter_gallery_assets';
 
+enum CardDemoType {
+  static,
```

Comment:
Consider picking a new name for `static` since `static` is a keyword in Dart.

Reply:
oops, done